utilities/bigquery.py

```diff
@@ -16,6 +18,27 @@ def get_likes(id_token):
         where
             json_value(data, '$.uid') = '{uid}'
     '''
-    results = db.query(query)
+    results = db.query(query, job_config=job_config)
     likes = [json.loads(row[0]) for row in results]
     return likes
+
+def get_song(id_token, time_of_day, likes):
+    """Get song recommendation for the logged in user."""
+    uid = auth.verify_id_token(id_token)['uid'] # pylint: disable=unused-variable
```

Comment:
why is this unused - should it be removed and added in later, when there's a user field to query by?